content/en/docs/started/workstation/minikube-linux.md

```diff
@@ -153,8 +160,10 @@ cd ${KF_DIR}
 kfctl apply -V -f ${CONFIG_URI}
```

Comment:
Lines 144-

```
The following example installs Kubeflow v1.0.0 under the `/root/kubeflow/v1.0` directory:
...
```

Have you tried with v1.1 by any chance? The docs could look fresher with a v1.1 example

Reply:
Have to try it.. let me do that in 1 hr and will ping you once done :)

Reply:
@alfsuse I mean it's probably not necessary, so it's just an idea. Some users may notice this or ask if minikube is compatible with the latest KF v1.1 release.

Thanks for this, I really appreciate it.

Reply:
I'll give a try anyway.. but I was thinking we may merge it the meantime in case I can open another PR.ù

Reply:
Good idea.